ocaml/xapi/xapi_observer_components.ml

```diff
@@ -90,9 +90,17 @@ let is_component_enabled ~component =
                  |> List.mem component
             )
             observers
-        with _ -> false
+        with e ->
+          D.log_backtrace ();
+          D.error "is_component_enabled(%s) inner got exception: %s"
```

Comment:
This is not an error, it should be a warning, same below

Reply:
fixed